Tasks/ANT/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -39,7 +39,7 @@
   "loc.input.label.jdkArchitecture": "JDK Architecture",
   "loc.input.help.jdkArchitecture": "Optionally supply the architecture (x86, x64) of the JDK.",
   "loc.messages.LocateJVMBasedOnVersionAndArch": "Locate JAVA_HOME for Java %s %s",
-  "loc.messages.FailedToLocateSpecifiedJVM": "Failed to find specified JDK version. Please make sure environment variable '%s' exists and is set to the location of a corresponding JDK.",
```

Comment:
If we are removing a `%s`, we should update the places where we use this resource string so they don't pass an argument for %s. (I don't know offhand if extra arguments are simply ignored or produce a warning/error.)

Reply:
My update to the error message is now using the argument again.